src/components/pages/TeachersPage/index.tsx

```diff
@@ -131,7 +131,7 @@ export default function TeachersPage({ classroomName }: ClassroomProps) {
   }
 
   return (
-    <main>
+    <main style={{padding: '16px'}}>
```

Comment:
```suggestion
    <main>
```

Reply:
For the padding here and in StudentsPage/index.tsx I don't see this on my local. It was added in the first commit and then removed in the second commit. If you checkout the branch on your local does it still show?